src/ui/components/post/post_listings.rs

```diff
@@ -5,7 +5,7 @@ use leptos::*;
 #[component]
 pub fn PostListings(#[prop(into)] posts: MaybeSignal<Vec<PostView>>) -> impl IntoView {
   view! {
-    <ul class="divide-y divide-neutral sm:overflow-y-auto min-h-0 h-fit sm:h-full mb-14 sm:mb-4">
+    <ul class="divide-y divide-neutral min-h-0 h-fit sm:h-full mb-14 sm:mb-4">
```

Comment:
So main change was just moving these overflows out into the main page then, gotcha.